data/2020_21/04_december/advent.yml

```diff
@@ -7,6 +7,7 @@ date:
   end: "2020-12-24"
 contestants:
   min: ss1
+  max: ss1
```

Comment:
podľa mňa skôr `null`, možno aj min by mohlo byť `null` ale keďže to má "stredoškolskú" úroveň, tak tam môžme nechať asi aj `ss1`